pkg/resources/reconciler.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 var (
-	OwnerLabelKey = integreatlyv1alpha1.SchemeGroupVersion.Group + "/installation-uid"
+	OwnerLabelKey = integreatlyv1alpha1.SchemeGroupVersion.Group + "/installation-uid" //
```

Comment:
@HubertStefanski What's the purpose of the trailing `//` here?

Reply:
@grdryn That one must've been added by accident :thinking: , will fix now